.github/workflows/macos-executable-release.yml

```diff
@@ -28,9 +28,36 @@ jobs:
         run: |
           git fetch --prune --unshallow
           echo "::set-output name=SHA_SHORT::$(git rev-parse --short HEAD)"
-  pack-macos:
+  set-file-path:
     needs: [get-tag, get-sha]
     runs-on: macos-11
+    outputs:
+     FILE_PATH_X64: ${{steps.set-file-path.outputs.FILE_PATH_X64}}
+     FILE_PATH_ARM64: ${{steps.set-file-path.outputs.FILE_PATH_ARM64}}
+    steps:
+     - uses: actions/checkout@v2
+     - name: Setting file path
+       id: set-file-path
+       run: |
+          if [[ ${{ needs.get-tag.outputs.TAG_NAME }} == *"rc"* ]]; then
+            file_path_x64=dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.${{ needs.get-sha.outputs.SHA_SHORT }}.pkg
+            file_path_arm64=dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.${{ needs.get-sha.outputs.SHA_SHORT }}-arm64.pkg
+          else
+            file_path_x64=dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.pkg
```

Comment:
[nit]: Can we have the string `-x64` in the filepath which is uploaded in Github assets. `dist/macos/twilio-v1.0.0-x64.pkg`